lib/ruby_llm/providers/bedrock/chat.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RubyLLM
+  module Providers
+    module Bedrock
+      # Chat methods for the AWS Bedrock API implementation
+      module Chat
+        module_function
+
+        def completion_url
+          "model/#{model_id}/invoke"
+        end
+
+        def model_id
+          @model_id
+        end
+
+        def render_payload(messages, tools:, temperature:, model:, stream: false)
+          @model_id = model
```

Comment:
Not a big fan of this, but I see why you did that. A comment would help other people not trip up on it!